common/app/views/fragments/topNav/servicesLinks.scala.html

```diff
@@ -21,7 +21,7 @@
             <a class="brand-bar__item--action" data-link-name="uk : topNav : soulmates"
                href="https://soulmates.theguardian.com/?INTCMP=NGW_TOPNAV_UK_GU_SOULMATES">dating</a>
         </div>
-        <div class="brand-bar__item has-popup brand-bar__item--has-control brand-bar__item--more">
+        <div class="brand-bar__item has-popup brand-bar__item--has-control brand-bar__item--more is-not-modern modern-visible">
```

Comment:
I thought is-not-modern was a marker added to the body element if we're in core mode?